backend/ibutsu_server/util/projects.py

```diff
@@ -15,3 +17,27 @@ def get_project_id(project_name):
     """Shorthand function for a repeated piece of code"""
     project = get_project(project_name)
     return str(project.id) if project else None
+
+
+def project_has_user(project, user):
```

Comment:
Should this method also check if the user is a project owner? 

I carried out the following steps:
1) Created a project where my user was the project owner
2) Selected the project from the project dropdown in the UI
3) Went to the dashboard/home page

The GET request on this page returns a 403 because `project_has_user` is false. This causes the UI to crash. We should handle the 403 on the FE to avoid the crash.

Reply:
I am still able to crash the frontend when a 403 from the `get_dashboard_list` is returned.

Reply:
Investigating this further, I actually don't think it's possible to get into this state without making code-modifications. Closing ...